### PR TITLE
optimize: seata-distribution add apm-seata-skywalking

### PR DESCRIPTION
**Pull request by zhaoyuguang**
fix https://github.com/seata/seata/issues/3830
package seata
export SKYWALKING_ENABLE=true 
run seata-server.sh

### CODE REVIEW

File: distribution/bin/seata-server.bat [hunk 1]
```diff
@@ -91,7 +91,7 @@ if exist %BASEDIR%/logs (
   md "%BASEDIR%/logs"
 )
```
Comment (slievrly) [inline distribution/bin/seata-server.bat:93]: if "%SKYWALKING_ENABLE%"=="true"

File: ext/apm-seata-skywalking-plugin/config/agent.config [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+# Kafka producer configuration
+# plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+# if you want to set namespace. please make sure the OAP server has set it in Kafka fetcher module
+# plugin.kafka.namespace=${SW_KAFKA_NAMESPACE:""}
+
+# Match spring bean with regex expression for classname
+# plugin.springannotation.classname_match_regex=${SW_SPRINGANNOTATION_CLASSNAME_MATCH_REGEX:}
+
+# If set to true, to identify the current node as Seata Server
+plugin.seata.server=${SW_SEATA_SERVER:true}
```
Comment (caohdgege) [inline ext/apm-seata-skywalking-plugin/config/agent.config:117]: the current config should be mark as unchangeable

File: distribution/bin/seata-server.bat (hunk 1 above)
Comment (zhaoyuguang) [reply distribution/bin/seata-server.bat:93]: fix, but without Window environment, to verify correctness

File: ext/apm-seata-skywalking-plugin/config/agent.config (hunk 2 above)
Comment (zhaoyuguang) [reply ext/apm-seata-skywalking-plugin/config/agent.config:117]: fix

**Review by slievrly — APPROVED** [review]
LGTM

**Review by caohdgege — APPROVED** [review]
LGTM

**Pull request merged**